src/magic_ter_furn_transform.h

```diff
@@ -21,10 +21,10 @@ class ter_furn_data
 {
     public:
         weighted_int_list<T> list;
-        std::string message;
+        std::optional<translation> message;
```

Comment:
`translation` already has a null state which can be tested by `empty()`, so `std::optional` is not needed here.